scrabbleScoreExercise/scrabbleScore.js

```diff
@@ -0,0 +1,69 @@
+// Programme: takes each letter of string input and saves it in an array
+//loops through each letter, saves corresponding score for the letter
+// scores are then summed up and scrabble score is displayed back to user
+'use strict'
+
+const score1 = ['a', 'e','i','o','u', 'l','n','r','s','t'];
+const score2 = ['d','g'];
+const score3 = ['b', 'c','m','p'];
+const score4 = ['f','h','v','w','y'];
+const score5 = ['k'];
+const score8 = ['j','x'];
+const score10 = ['q','z'];
```

Comment:
@NosiphoManyase awesome solution! JS code is looking good!
 another solution you could play with would be creating an object for your scores like this:

```
const letter_scores_mapping = {
     1: ['a', 'e','i','o','u', 'l','n','r','s','t'],
     2: ['d','g'],
     3: ['b', 'c','m','p']
}
```

You would then not need to create each `const numberScore` below.

Reply:
That came across my mind, but I wasn't sure how to loop through each property in an object and check for the presence of the letter. But I'll try and look into that.

Reply:
You're right - it's actually quite a difficult structure to iterate over. 

Another one would be to map each letter to it's score:

``` 
const letter_scores_mapping = {
     a: 1
     b: 3
     c: 3
     ...
}
```

With this one it's easier to reference the values in the object, but just more typing to set the data structure up :D